resources/api/reservation.py

```diff
@@ -378,6 +379,44 @@ def filter_queryset(self, request, queryset, view):
         return queryset
 
 
+class ReservationFilterSet(django_filters.FilterSet):
```

Comment:
This should probably derive from `django_filters.rest_framework.FilterSet` (and then I think the `DRFFilterBooleanWidget` hack wouldn't be needed)?